src/functions.jl

```diff
@@ -14,13 +14,27 @@ julia> ds = KeyedDataset(
            :val2 => KeyedArray(ones(3, 2); time=1:3, loc=[:x, :y]),
        );
 
-julia> mean.((ds.val1, ds.val2))
-(0.0, 1.0)
-
 julia> r = map(a -> a .+ 1.0, ds; dims=:loc);
 
-julia> mean.((r.val1, r.val2))
-(0.0, 2.0)
+julia> r[:val1]
+2-dimensional KeyedArray(NamedDimsArray(...)) with keys:
+↓   time ∈ 3-element ReadOnlyArray{Int64,...}
+→   obj ∈ 2-element ReadOnlyArray{Symbol,...}
+And data, 3×2 Array{Float64,2}:
+      (:a)  (:b)
+ (1)   0.0   0.0
+ (2)   0.0   0.0
+ (3)   0.0   0.0
+
+julia> r[:val2]
+2-dimensional KeyedArray(NamedDimsArray(...)) with keys:
```

Comment:
Fixing this is going to be annoying cause doctests are really finicky about both whitespace and when internal module names are used. Also, using `fix=true` only fixes some issues.

Reply:
Dropped it in favour of just showing the parent arrays.